### PR TITLE
test(#429): complete assertion for test and rename test

### LINKED ISSUE #429: complete assertion for ChangeArquillianConfigurationPreparerIT
### Description
Getting Code smell warning for Sonarcloud due to incomplete assertion

### What should happen?
should not get code smell for ChangeArquillianConfigurationPreparerIT

### What happens instead?
Getting code smell for ChangeArquillianConfigurationPreparerIT

### Do you have any idea why?
Incomplete assertion

### PR DESCRIPTION
**Pull request by dipak-pawar**
### Why
#429

### Description

<Please describe your changes here.>

Closes #429

### Checklist

- [x] I followed the contribution [guidelines](https://raw.githubusercontent.com/fabric8-launcher/launcher-backend/master/CONTRIBUTING.md).
- [x] I created an [issue](https://github.com/fabric8-launcher/launcher-backend/issues/429) and used it in the commit message.
- [x] I have built the project locally prior to submission with `mvn clean install`.
- [x] I kept a clean commit log (no unnecessary commits, no merge commits).
- [x] I am happy with my contribution.

### CODE REVIEW

File: core/core-impl/src/test/java/io/fabric8/launcher/core/impl/preparers/ChangeArquillianConfigurationPreparerIT.java [hunk 1]
```diff
@@ -88,11 +88,11 @@ public void shouldUpdateArquillianConfiguration() throws IOException, ExecutionE
       // then
       assertThat(
          contentOf(Paths.get(rootPath.toString(), "src/test/resources/arquillian.xml").toFile())
-            .contains("<property name=\"app.name\">my-artifact-id</property>"));
```
Comment (gastaldi) [inline core/core-impl/src/test/java/io/fabric8/launcher/core/impl/preparers/ChangeArquillianConfigurationPreparerIT.java]: That works, but I wonder if it isn't easier to do `assertThat(contentOf(...)).contains(...)`?

Comment (dipak-pawar) [reply core/core-impl/src/test/java/io/fabric8/launcher/core/impl/preparers/ChangeArquillianConfigurationPreparerIT.java]: I think this is only easier way with which we can check content of file

Comment (gastaldi) [reply core/core-impl/src/test/java/io/fabric8/launcher/core/impl/preparers/ChangeArquillianConfigurationPreparerIT.java]: The error message is more descriptive using the suggestion above, but I am totally fine if you want to keep like that

Comment (dipak-pawar) [reply core/core-impl/src/test/java/io/fabric8/launcher/core/impl/preparers/ChangeArquillianConfigurationPreparerIT.java]: If I used `assertThat(contentOf(...)).contains(...)` on single line then user has to scroll to check entire line, this is why I split it into multiple line

**Pull request merged**